moveit_plugins/moveit_ros_control_interface/src/controller_manager_plugin.cpp

```diff
@@ -109,6 +110,8 @@ class MoveItControllerManager : public moveit_controller_manager::MoveItControll
   rclcpp::Client<controller_manager_msgs::srv::ListControllers>::SharedPtr list_controllers_service_;
   rclcpp::Client<controller_manager_msgs::srv::SwitchController>::SharedPtr switch_controller_service_;
 
+  std::unordered_map<std::string, std::vector<std::string>> dependency_map_;
```

Comment:
Does this only apply to chained controllers? May want to rename it `chained_dependencies_map_` or something similar, if so. And add comments.

```suggestion
  // Chained controllers have dependencies (other controllers which must be running)
  std::unordered_map<std::string /* controller name */, std::vector<std::string> /* dependencies */> dependency_map_;
```

Reply:
It does only apply to chained controllers for now, so the name should be changed.